1-Basic/UserDetailsClient/UserDetailsClient.Droid/UserDetailsClient.Droid.csproj

```diff
@@ -42,6 +42,7 @@
     <AndroidEnableProfiledAot>false</AndroidEnableProfiledAot>
     <BundleAssemblies>false</BundleAssemblies>
     <AndroidEnableMultiDex>true</AndroidEnableMultiDex>
+    <AndroidSupportedAbis>arm64-v8a;x86;armeabi-v7a</AndroidSupportedAbis>

```

Comment:
Is this needed?

Reply:
This is to test on arm phones. Else developer will have to modify it.